python/mdvtools/dbutils/mdv_server_app.py

```
@@ -1,10 +1,13 @@
 import os
 import time
 import json
+import shutil
```

Ruff (0.8.2)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)